From 2e7da42120f0dd9244e5383b28e5f61f77f20529 Mon Sep 17 00:00:00 2001
From: Ashish Singhi <ashish.singhi@huawei.com>
Date: Mon, 22 Dec 2014 14:13:46 +0530
Subject: [PATCH] HBASE-12511 namespace permissions - add support from table
 creation privilege in a namespace 'C'. Addendum to fix java doc warning

---
 .../java/org/apache/hadoop/hbase/security/access/AccessController.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hbase-server/src/main/java/org/apache/hadoop/hbase/security/access/AccessController.java b/hbase-server/src/main/java/org/apache/hadoop/hbase/security/access/AccessController.java
index 98ee719..48464f6 100644
--- a/hbase-server/src/main/java/org/apache/hadoop/hbase/security/access/AccessController.java
+++ b/hbase-server/src/main/java/org/apache/hadoop/hbase/security/access/AccessController.java
@@ -539,7 +539,7 @@ public class AccessController extends BaseMasterAndRegionObserver
   /**
    * Checks that the user has the given global or namespace permission.
    * @param namespace
-   * @param perm Action being requested
+   * @param permissions Actions being requested
    */
   public void requireNamespacePermission(String request, String namespace,
       Action... permissions) throws IOException {
-- 
1.9.2.msysgit.0

